src/main/webapp/app/guided-tour/guided-tour.component.ts

```diff
@@ -150,7 +150,7 @@ export class GuidedTourComponent implements AfterViewInit, OnDestroy {
                 this.scrollToAndSetElement();
                 this.handleTransition();
                 this.guidedTourService.isBackPageNavigation.next(false);
-                if (this.currentStepIndex && this.nextStepIndex) {
+                if (this.currentStepIndex !== undefined && this.nextStepIndex !== undefined) {
```

Comment:
this is to make sure that if the index is 0 (which is often the case) that the expression can still evaluate to true. In ts 0 is falsy.